AdminWebsite/AdminWebsite/ClientApp/src/app/work-allocation/edit-work-hours/vho-work-hours-table/vho-work-hours-table.component.ts

```diff
@@ -16,22 +17,30 @@ export class ValidationFailure {
 export class VhoWorkHoursTableComponent implements CanDeactiveComponent {
     constructor(private videoHearingsService: VideoHearingsService) {}
 
-    @Input() set result(value) {
-        if (value && value[0] instanceof VhoWorkHoursResponse) {
+    @Input() set result(value: VhoWorkHoursResponse[]) {
+        if (value) {
             this.workHours = value;
         } else {
             this.workHours = null;
         }
+        this.checkVhoHasNWorkHoursToEdit();
     }
-
     public static readonly ErrorStartAndEndTimeBothRequired = 'Both Start Time and End Time must be filled in or empty';
     public static readonly ErrorEndTimeBeforeStartTime = 'End Time cannot be before Start Time';
+    public static readonly WarningNoWorkingHoursForVho =
+        'There are no working hours available to edit for this user. Please upload this users working hours before they can be edited.';
```

Comment:
missing an apostrophe here. Please upload this **user's** working hours

Reply:
this fails the lint checks can you please suggest what to include for the lint.

Reply:
add the apostrophe back in, and then you can add this line above that message to tell lint to ignore it:

`// tslint:disable-next-line: quotemark`